server/asset/btc/rpcclient.go

```diff
@@ -0,0 +1,142 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package btc
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/btcsuite/btcd/btcjson"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcutil"
+)
+
+const (
+	methodGetBestBlockHash  = "getbestblockhash"
+	methodGetBlockchainInfo = "getblockchaininfo"
+	methodEstimateSmartFee  = "estimatesmartfee"
+	methodGetTxOut          = "gettxout"
+	methodGetRawTransaction = "getrawtransaction"
+	methodGetBlock          = "getblock"
+)
+
+// RawRequester defines dcrd's rpcclient RawRequest func where all RPC
+// requests sent through. For testing, it can be satisfied by a stub.
```

Comment:
> RawRequester defines dcrd's rpcclient RawRequest func

The interface doesn't **define** rpcclient's `RawRequest`, and `RawRequest` is not a function. 

How about "RawRequester is for sending context-aware RPC requests, and has methods for shutting down the underlying connection."

Reply:
Yep, reads much better